types/repo/AnonymousMessageRepo/AnonymousPostNotificationData.ts

```diff
@@ -19,6 +19,10 @@ export interface PostMaker {
 export interface PostMakerData {
   profile_pic_url?: string;
   username: string;
+  emoji_name: string | null;
+  emoji_code: string | null;
+  color_name: string | null;
+  color_code: string | null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `emoji_name`, `emoji_code`, `color_name`, and `color_code` to the `PostMakerData` interface seems appropriate for storing anonymous user information. However, it's important to ensure that these fields are properly validated before being used in the application. For instance, `emoji_code` and `color_code` should be checked to ensure they represent valid codes, while `emoji_name` and `color_name` should be sanitized to prevent potential security issues such as XSS attacks.
```diff
 export interface PostMakerData {
   profile_pic_url?: string;
   username: string;
+  emoji_name: string | null;
+  emoji_code: string | null;
+  color_name: string | null;
+  color_code: string | null;
 }
```
Please make sure to add appropriate validation for these new fields in the part of your code where you handle this data.


<!-- This is an auto-generated comment by OSS CodeRabbit -->